src/app/pages/getting-started/getting-started.html

```diff
@@ -26,6 +26,7 @@ <h2>Dependencies</h2>
                 <td>Font Awesome</td>
                 <td>IE11 support</td>
                 <td>Microsoft Edge (pre-chromium) support</td>
+                <td>LCP features</td>
```

Comment:
probably, this is too much, because I cannot add a column for every new feature.
I created this table to show dependencies and browsers, not for specific features.